awx/sso/middleware.py

```diff
@@ -35,33 +31,14 @@ def process_request(self, request):
             request.successful_authenticator = None
 
         if not request.path.startswith('/sso/') and 'migrations_notran' not in request.path:
-
```

Comment:
Do we want to fully remove this now, or simply deprecate it?

Reply:
Fully remove it, these code bring tracebacks after the auth renewal.